R/pkg/R/functions.R

```diff
@@ -206,6 +206,8 @@ NULL
 #' df <- createDataFrame(cbind(model = rownames(mtcars), mtcars))
 #' tmp <- mutate(df, v1 = create_array(df$mpg, df$cyl, df$hp))
 #' head(select(tmp, array_contains(tmp$v1, 21), size(tmp$v1)))
+#' head(select(tmp, array_max(tmp$v1)))
+#' head(select(tmp, array_min(tmp$v1)))
```

Comment:
let's combine these into one line? I think multiple examples in the pattern of "head(select(...))" isn't super useful to be honest